crates/bevy_ui/src/ui_node.rs

```diff
@@ -123,14 +161,21 @@ impl Default for AlignItems {
     }
 }
 
+/// Works like [`AlignItems`] but applies only to a single item
 #[derive(Copy, Clone, PartialEq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect_value(PartialEq, Serialize, Deserialize)]
 pub enum AlignSelf {
+    /// Use the value of [`AlignItems`]
     Auto,
+    /// If the parent has [`AlignItems::Center`] only this item will be at the start
```

Comment:
Why are you mentioning Center in particular?

Reply:
The descriptions are heavily inspired by https://cssreference.io/flexbox/#align-items, which uses that as an example to underline how this causes the item to diverge from the parent's align-items property